packages/dd-trace/src/appsec/iast/taint-tracking/plugin.js

```diff
@@ -45,7 +45,7 @@ class TaintTrackingPlugin extends SourceIastPlugin {
     this.addSub(
       { channelName: 'apm:express:middleware:next', tag: HTTP_REQUEST_BODY },
       ({ req }) => {
-        if (req && req.body && typeof req.body === 'object') {
+        if (req && req.body !== null && typeof req.body === 'object') {
```

Comment:
optional chaining at this point

Reply:
Since I was making so many edits, I wanted to keep the rewriting to a minimum, otherwise it quickly spirals out of control 😅